### PR TITLE
Fix the directory where gems are packaged by bundle

### PR DESCRIPTION
**Pull request by Ana06**
`vendor/cache` instead of `vendor/bundle/vendor/cache`. :bowtie: 

Co-authored-by: Björn Geuken <bgeuken@suse.de>

### CODE REVIEW

**Comment by jordimassaguerpla** [comment]
Hi @Ana06 : Could you be a bit more verbose :) ?

Which issue was this causing?
Will your change break existing packages?

**Comment by bgeuken** [comment]
> jordimassaguerpla commented 15 hours ago
> Hi @Ana06 : Could you be a bit more verbose :) ? Which issue was this causing? Will your change break existing packages?

Yes, this would break packages that use this service and rely on vendor/bundle/vendor/cache

@Ana06 We forgot to update [this line](https://github.com/Ana06/obs-service-bundle_gems/blob/12762e09df864e20da5da0dd77b702dee4f75d08/bundle_gems#L107) and the [Readme](https://github.com/openSUSE/obs-service-bundle_gems/blob/master/README.md#cpio-mode) also needs an update.

**Comment by jordimassaguerpla** [comment]
I would prefer we don't break existing packages. Is this fixing an issue or it is just that you don't  like the path?

**Comment by Ana06** [comment]
@bgeuken 

> @Ana06 We forgot to update this line and the Readme also needs an update.

nice that I read you comment now... after debugging it and finding it out already :rofl:

**Comment by bgeuken** [comment]
> jordimassaguerpla commented 9 hours ago
> I would prefer we don't break existing packages. Is this fixing an issue or it is just that you don't  like the path?

Good point. We could add a symlink to keep existing packages working (vendor/bundle/vendor/cache -> vendor/cache/)

**Review by jordimassaguerpla — DISMISSED** [review]
Make sure this does not break existing packages. A symlink to the old directory may work, but needs to be tested.

**Comment by ChrisBr** [comment]
> Make sure this does not break existing packages. A symlink to the old directory may work, but needs to be tested.

I'm wondering if we really want to already carry around legacy code for mistakes we did ~4 weeks ago (on build.o.o we deployed it ~3 weeks ago)  :cry: I would even claim that the :cpio mode is still in 'beta' and I assume the obs-server package is currently the only user (although I don't know of course). 

So I really would like to think about if this is really needed. It is not only a symlink, it is also necessary to adapt the find etc etc. Not to speak from the 'obfuscation' of the code...

TL;DR: Yes this change will break existing packages but I'm not sure if we really need to be backward compatible given that the code is > 1 month in production. We could mention it in our next OBS sprint report to make people aware of the change.

cc @jordimassaguerpla @bgeuken

**Comment by jordimassaguerpla** [comment]
@ChrisBr :  What you say also makes sense. Ok with fixing it without backward compatibility.

**Comment by Ana06** [comment]
@jordimassaguerpla can we merge it then? :pray:

**Comment by Ana06** [comment]
@jordimassaguerpla can we merge it then? :pray:

**Comment by Ana06** [comment]
:tada:
